### DISCUSSION TITLE
Ratio for uSv

### DISCUSSION THREAD

**Discussion by sonny1985** [steadramon/ESPGeiger#14] [General]
Hello :) First i wanna say thanks alot for this great Project.
I use it , because of the included MQTT i can work and play around a bit in iobroker with it :)

Programmed one of my ESP´s already for it. Im getting the Geiger Board RadiationD V1.1 (CAJOE), what i ordered on Amazon, tomorrow.

I have a question about the setting: **"Ratio for uSv Calculation"**.
Default it looks like its 151 .0

**Does this Value depend on the Geiger Tube you are using ? in my case, this Board comes with a J305 Tube.**

Can i leave this Value on 151.0 or is there any change neccesary ?
Model value i assigned "CaJoe".

Greetings from Austria :)

**Comment by steadramon** [comment]
Hi! Hope your Cajoe arrived safe and sound.

The matter of conversion rates etc is discussed around the internet and I think one of the best writeups is here - https://iot-devices.com.ua/en/geiger-tube-j305-how-to-calculate-the-conversion-factor-of-cpm-technical-note-en

The answer really "depends" - there are two values suggested above - however I believe these are based upon the longer J305 (105mm) - the Cajoe you get will likely have the shorter J305 at 90mm.

Two numbers are mentioned on the page above -

> The first is the one we calculated based on the data in the datasheet: 0.00332.
> The second is the one that everyone uses: 0.00812

`1 / 0.00332` = **301.2**
`1 / 0.00812` = **123.2**

Here is it is listed as "~0.0081" - again this is the longer 105mm±

https://www.rhelectronics.store/j305-glassy-geiger-muller-tube-nuclear-radiation-sensor

As pointed out in the article above, the ratios above will make a big difference to the calculated uSV.

The default of 151.0 could be an OK starting point - 123 seems to be the value "people use". I currently have an M4011 tube in my CaJoe set up so I can't help much with suggested values.

A great place to ask about good ratios for different tubes would be the experts over at [radmon.org](https://radmon.org)

**Comment by sonny1985** [comment]
Hi. Thanks alot for your Information.(Im learning about this things)
This Counter arrives today :) I will install it immediately.

On the Amazon "Reviews" i found some Information:

There is also other confusion involved. I will post a pic of some of the popular geiger/Muller tube info data. It comes with a J305 tube. If you replace it with a Russian tube make sure to calculate the rate according to their data sheet. This is the correct conversion for the j305 : 0.0038 = strait conversion
301.03 = 0.0033 compensated for absorbtion.

It looks like this Guy found out the correct specifications for this 90mm Tube.
Now i have 2 Values here:

1/0.0038 = 263.15 (Strait)
1/0.0033 = 303,03 (Absorption)
one of them will Fit perfect to this Tube i think. I will just try and see. I will get myself some Uranglas (Its radiation is very low and dont harm anything - But its definetly measureable with the Geiger Counters: Here you see a Video in the middle of the page: https://www.radonshop.com/waltec-uranglas-testquelle-fuer-geigerzaehler )
Just for fun and experimenting a bit :)

![51yFFWgkDJL](https://github.com/user-attachments/assets/528d8bc9-eb71-44e8-af46-df4afbdc1f2f)
